GitUI/CommandsDialogs/SettingsDialog/Pages/AdvancedSettingsPage.Designer.cs

```diff
@@ -141,14 +141,15 @@ private void InitializeComponent()
             this.tableLayoutPanel1.Controls.Add(this.chkDontSHowHelpImages, 0, 0);
             this.tableLayoutPanel1.Controls.Add(this.chkConsoleEmulator, 0, 4);
             this.tableLayoutPanel1.Location = new System.Drawing.Point(6, 21);
+            this.tableLayoutPanel1.Dock = System.Windows.Forms.DockStyle.Fill;
             this.tableLayoutPanel1.Name = "tableLayoutPanel1";
             this.tableLayoutPanel1.RowCount = 5;
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
-            this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Absolute, 20F));
             this.tableLayoutPanel1.Size = new System.Drawing.Size(361, 120);
+            this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.TabIndex = 1;
```

Comment:
Thanks. WinForms designer more often harms than helps, didn't notice that it has hardcoded the row height at some weird value. Opening on machines with 96 and 192 DPIs does strange things to it.